config/kubernetes-csi/org.yaml

```diff
@@ -330,3 +330,21 @@ teams:
     - lpabon
     - saad-ali
     privacy: closed
+  csi-misc:
```

Comment:
nit: can we have this in the alphabetical order? all team names are currently listed alphabetically 

(i'll add a test for this later)

Reply:
Fixed